Symfony/CS/ConfigurationResolver.php

```diff
@@ -84,6 +99,10 @@ public function getFixers()
 
     public function getProgress()
     {
+        if (!method_exists($this->config, 'getHideProgress')) {
```

Comment:
it will never happen
https://github.com/rybakit/PHP-CS-Fixer/blob/config_interface/Symfony/CS/ConfigurationResolver.php#L54